NitroxClient/Communication/Packets/Processors/EntityDestroyedProcessor.cs

```diff
@@ -24,27 +24,25 @@ public EntityDestroyedProcessor(Entities entities, IPacketSender packetSender)
     public override void Process(EntityDestroyed packet)
     {
         entities.RemoveEntity(packet.Id);
+        if (!NitroxEntity.TryGetObjectFrom(packet.Id, out GameObject gameObject))
+        {
```

Comment:
Can probably add some logging in case it happens

Reply:
@Measurity